src/Compiler/FSharp.Compiler.Service.fsproj

```diff
@@ -27,6 +27,11 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
 
+  <!--  The FSharp.Compiler.Service dll provides a referencable public interface for tool builders -->
+  <PropertyGroup Condition="'$(Configuration)' != 'Proto'">
+      <CompressMetadata>true</CompressMetadata>
```

Comment:
This doesn't look correct. Should this be false unconditionally?  That is this component never uses compression?

Reply:
Individual projects correct it.  By and large FSharp.Core is the only one we want compressed eventually.  But we are in a chicken and egg situation.

However, tomorrows Preview vs will support compression fine.